src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -96,60 +98,66 @@ class AddSecondaryLoginPage extends Component {
     render() {
         return (
             <ScreenWrapper>
-                <HeaderWithCloseButton
-                    title={this.formType === CONST.LOGIN_TYPE.PHONE ? 'Add Phone Number' : 'Add Email Address'}
```

Comment:
This is not how platform-specific code should be done. Please refer to the readme because there is a section there with details.